src/main/java/org/generationcp/middleware/api/location/LocationService.java

```diff
@@ -38,4 +38,10 @@
 	LocationDTO createLocation(LocationRequestDto locationRequestDto);
 
 	void updateLocation(Integer locationId, LocationRequestDto locationRequestDto);
+
+	boolean existsLocationAsCountry(Integer locationId);
```

Comment:
@cuenyad rename to isDefaultCountryLocation

Reply:
done.